src/components/GrampsjsFormEditPlaceName.js

```diff
@@ -5,9 +5,22 @@ Form for adding a new event reference
 import {html} from 'lit'
 
 import './GrampsjsFormString.js'
+import './GrampsjsFormSelectDate.js'
 import {GrampsjsObjectForm} from './GrampsjsObjectForm.js'
 
 class GrampsjsFormEditPlaceName extends GrampsjsObjectForm {
+  static get properties() {
+    return {
+      ...super.properties,
```

Comment:
There's need for that, see [Lit docs](https://lit.dev/docs/components/properties/): 

> You don't need to redeclare properties unless you want to change options.